fix rubocops

kbrock opened the pull request:
Still have a bunch of rubocops in the spec due to the way we create the `TestClass`.
But this should resolve most other issues.
<!--
1. Describe what this Pull Request does and why you think it is needed.
   If this PR includes UI or CLI changes, please include Before/After screenshots
   If this PR includes performance changes, please include Before/After metrics showing improvement.
-->

<!--
2. If this fixes an existing issue, please specify in `Fixes #<id>` format
   (As described in https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
-->

<!--
3. Ask @miq-bot to apply a scope label (bug, enhancement, etc) and any additional reviewers or assignees.
   (As described in https://github.com/ManageIQ/miq_bot#requested-tasks)
   e.g. `@miq-bot add-label label_name`
        `@miq-bot add-reviewer @name`
-->

Fryguy commented on lib/active_record/virtual_attributes/virtual_total.rb:
```diff
@@ -116,4 +116,4 @@ def virtual_aggregate_arel(reflection, method_name, column)
 end
 
-ActiveRecord::Base.include VirtualAttributes::VirtualTotal
+ActiveSupport.on_load(:active_record) { include VirtualAttributes::VirtualTotal }
```
Is the block in the scope of ActiveRecord::Base?  (How does this work?)

kbrock replied:
ooh. good point. That is what Rubocop asked me to change, but I see what you mean.

Not totally sure how this works, I'm going to roll this back.

kbrock commented:
update:
- rolled back `ActiveSupport.on_load(:active_record)`

- update:
-  added `Rails/ApplicationRecord` ignores
- dropped `Naming/PredicateMethod` ignores
